app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/Shipments.php

```diff
@@ -26,14 +26,17 @@ class Shipments extends \Magento\Framework\View\Element\Text\ListText implements
      *
      * @param \Magento\Framework\View\Element\Context $context
      * @param \Magento\Framework\Registry $coreRegistry
+     * @param \Magento\Framework\AuthorizationInterface|null $authorization
      * @param array $data
      */
     public function __construct(
         \Magento\Framework\View\Element\Context $context,
         \Magento\Framework\Registry $coreRegistry,
+        \Magento\Framework\AuthorizationInterface $authorization = null,
```

Comment:
New constructor parameters should be at the end of the list (after $data in this case)